snippets/multi-arch-schedule-toleration.adoc

```diff
@@ -0,0 +1,25 @@
+:_mod-docs-content-type: SNIPPET
+
+Tolerating architecture taints in workloads:: When a node or machine set has a taint, only workloads that tolerate that taint can be scheduled. You can configure your workload with a `toleration` so that it is scheduled on nodes with specific architecture taints.
++
+.Example deployment with toleration set
+--
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata: # ...
+spec:
+  # ...
+  template:
+    # ...
+    spec:
+      tolerations:
+      - key: "multiarch.openshift.io/arch"
+        value: "arm64"
+        operator: "Equal"
+        effect: "NoSchedule"
+----
++
```

Comment:
This `+` is getting rendered: 
![image](https://github.com/user-attachments/assets/f3551a4e-2885-4029-8462-56f91ed6019a)

Reply:
Ah, thanks for the catch! Will update this today

Reply:
Fooled by the use of the `--` block here :) thanks again!